libraries/package-deps-hash/src/getPackageDeps.ts

```diff
@@ -217,6 +223,8 @@ export function gitStatus(path: string, gitPath?: string): string {
   );
 
   if (result.status !== 0) {
+    ensureGitMinimumVersion(gitPath);
```

Comment:
Did you check that `ensureGitMinimumVersion()` is never called more than once for a given invocation of the Rush CLI?  (Is there some way we could guarantee that?)

Reply:
I can memoize the `git version` result.